src/main/java/com/sumindaa/demo/employeeservice/service/EmployeeService.java

```diff
@@ -0,0 +1,20 @@
+package com.sumindaa.demo.employeeservice.service;
```

Comment:
I need the implementation of this. You no need to use a database. You can use Simple Map, Set or List whatever you think suits best.